src/commands/ios_build.yml

```diff
@@ -1,6 +1,10 @@
 description: Builds the iOS app at the given path with the given build scheme and configuration. This should be run only after installing NPM dependencies.
 
 parameters:
+  pod_install_directory:
+    type: string
+    default: ""
```

Comment:
I don't think that the "build" command should run `pod install`. I think that running that is more part of a "setup" step.

If users are using the commands directly, then they can just run the `pod_install` command before this one. It's more obvious than having to set a parameter to get the desired behaviour.

Reply:
makes sense I removed all changes made to this file